src/conversions/chrono.rs

```diff
@@ -223,17 +306,32 @@ impl<Tz: TimeZone + for<'a> FromPyObject<'a>> FromPyObject<'_> for DateTime<Tz>
 impl ToPyObject for FixedOffset {
     fn to_object(&self, py: Python<'_>) -> PyObject {
         let seconds_offset = self.local_minus_utc();
-        let td =
-            PyDelta::new(py, 0, seconds_offset, 0, true).expect("failed to construct timedelta");
-        timezone_from_offset(py, td)
-            .expect("Failed to construct PyTimezone")
-            .into()
+
+        #[cfg(not(Py_LIMITED_API))]
+        {
+            let td = PyDelta::new(py, 0, seconds_offset, 0, true)
+                .expect("failed to construct timedelta");
+            timezone_from_offset(py, td)
+                .expect("Failed to construct PyTimezone")
+                .into()
+        }
+        #[cfg(Py_LIMITED_API)]
+        {
+            let td = Duration::seconds(seconds_offset.into()).into_py(py);
+            static TIMEZONE: GILOnceCell<Py<PyType>> = GILOnceCell::new();
```

Comment:
I think I would prefer to handle this uniformly using a helper function as for the other type objects.

I also wonder whether it would make sense to load all of these types from `datetime` in a single pass, i.e. put a struct containing multiple `Py<PyType>` fields into a single `GILOnceCell` to reduce the overhead and collect these into a single place?

Reply:
Done.